auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.kt

```diff
@@ -1017,6 +1073,13 @@ public class AuthenticationAPIClient @VisibleForTesting(otherwise = VisibleForTe
             factory.post(url.toString(), credentialsAdapter), clientId, baseURL
         )
         request.addParameters(requestParameters)
+        try {
```

Comment:
Should we extract this logic into a helper to avoid copy-pasting the same block many times?